src/lib.rs

```diff
@@ -27,10 +116,14 @@ pub fn metadata() -> &'static cargo_metadata::Metadata {
     &METADATA
 }
 
+/// Allows you to fetch informations of a package in the current crate
```

Comment:
There is no plural for the word "information". Check your spell checker is enabled.

```suggestion
/// Fetch information of a package in the current crate.
```